plugins/wazo-yealink/v86/common.py

```diff
@@ -274,6 +278,8 @@ def _nb_linekey_by_model(self, model):
     def _expmod_by_model(self, model):
         if model in ('T27G', 'T46S', 'T48S'):
             return self.EXP40ExpansionModule
+        elif model in (u'T43U', u'T46U', u'T48U'):
```

Comment:
We are now in Python3. No more `u`! :smile:

Reply:
@bloom1 thank's, fixed.